src/election.jl

```diff
@@ -172,17 +195,18 @@ end
 
 
 function ElectionTracker(election::Election,
-                         partisan_metrics::Array{S, 1}=AbstractScore[])::CompositeScore where {S <: AbstractScore}
+                         scores::Array{S, 1}=AbstractScore[])::CompositeScore where {S <: AbstractScore}
     """ The ElectionTracker method returns a CompositeScore that first updates
         the vote count / share for changed districts and then proceeds to
-        calculate other partisan metrics, as desired by the user.
+        run other scores (such as vote count for a particular party, partisan
+        metrics, etc.), as desired by the user.
         Re-calculating vote counts only for changed districts means that the
         CompositeScore does not perform redundant computations for all of the
-        partisan metrics. Furthermore, packaging all partisan metrics within
-        the CompositeScore ensures that the vote update occurs first, followed
-        by the partisan metrics scoring functions.
+        partisan metrics. Furthermore, packaging all election-related scores
+        within the CompositeScore ensures that the vote update occurs first,
+        followed by the partisan metrics scoring functions.
     """
-    vote_tracker = [count_votes("votes", election)]
-    scores = Array{AbstractScore, 1}([vote_tracker; partisan_metrics])
+    count_votes = vote_updater("votes", election) # name does not matter
```

Comment:
Note: name doesn't matter because I've also updated `score_initial_partition` and `score_partition_from_proposal` to only store the results of score functions that return an actual value (i.e., they do not return `nothing`). Since the `vote_updater` function's purpose is only to update the `Election` object (not to actually return a value of a score), it will not appear in the object returned by `flip_chain()`/`recom_chain()`, and therefore the name we provide here has no real significance.